includes/assets.php

```diff
@@ -265,6 +265,37 @@ public function register_scripts() {
 				)
 			);
 
+			wp_add_inline_script(
+				'wp-blocks',
+				'const originalBlocks = window.wp.blocks;
+
+window.wp.blocks = new Proxy(originalBlocks, {
+  get(target, prop, receiver) {
+    if (prop === "getBlockBindingsSources") {
+      return function(...args) {
+        const result = target.getBlockBindingsSources(...args);
+		if (result?.["acf/field"]) {
+				result["acf/field"]["getFieldsList"] = function() {
+					return acf.getFields().reduce( (acc, { data, $el }) => {
+						acc[data.name] = {
+							label: $el.find("label").text(),
+							value: $el.find("input").val() || "",
```

Comment:
I guess this won't work for fields different that the text one. (For example the textarea).